test/e2e/data/infrastructure-docker/cluster-template-kcp-adoption.yaml

```diff
@@ -1,179 +1,180 @@
-##
-# these resources are sequenced by label:
-# 1. initial: just the resources to bootstrap an initial controlplane
-# 2. kcp: the KCP resources (note the duplicated Cluster to associate the control plane)
-##
-###
-# 1. initial
-###
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-kind: DockerCluster
+apiVersion: v1
```

Comment:
I would like to gitignore the generated cluster-templates and keep under source control only the bases, but it seems that this makes the apidiff batch to fail 🤔